api/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+go mod tidy
+air
```

Comment:
```suggestion
air

```
ファイルの一番最後の行には空行の追加をお願いします